pypureclient/flashblade/FB_2_8/models/file_lock.py

```diff
@@ -37,7 +37,7 @@ class FileLock(object):
         'inode': 'int',
         'path': 'str',
         'protocol': 'str',
-        'range': 'FilelockRange',
+        'range': 'FileLockRange',
```

Comment:
Please be sure to release-note this class name fix when making the 1.35.0 release.

Also, it seems strange that `file_lock_range.py` is being added with the new class name, but there doesn't seem to be a `filelock_range.py` being deleted... 🤔 
I don't think it would cause issues, but I would've expected it to be getting removed here.

Reply:
Update, it's now visible as [file rename](https://github.com/PureStorage-OpenConnect/py-pure-client/pull/60/files#diff-6f0d46017136cb72178a8e56699543083112e7efa3f12928e797a42d22f23b2b).

Adding the misc section following to release notes:
###     Misc:
- Fixing class name (`FilelockRange` → `FileLockRange`)
See: get_file_systems_locks